modular_skyrat/master_files/code/datums/traits/negative.dm

```diff
@@ -70,8 +70,8 @@
 	name = "Mute"
 	desc = "Due to some accident, medical condition, or simply by choice, you are completely unable to speak."
 	value = -2 //HALP MAINTS
-	gain_text = "<span class='danger'>You find yourself unable to speak!</span>"
-	lose_text = "<span class='notice'>You feel a growing strength in your vocal chords.</span>"
+	gain_text = "<span class = 'danger'>You find yourself unable to speak!</span>"
+	lose_text = "<span class = 'notice'>You feel a growing strength in your vocal chords.</span>"
```

Comment:
Isn't there a way to make this use span macros?

Reply:
Not sure tbh, I'd have to look into it.

Reply:
Looked into it and it works fine idk why it's not done anywhere on /tg/